examples/langmuir_turbulence.jl

```diff
@@ -273,14 +276,9 @@ iterations = parse.(Int, keys(fields_file["timeseries/t"]))
 
 function nice_divergent_levels(c, clim; nlevels=21)
```

Comment:
Will it look better with `nlevels=21`?

Reply:
I think we want odd levels for divergent quantities so that 0 is included ?

Reply:
Yes, but that way nothing shows as white... while if you have even number of levels then there is a small band around zero which is white... (see, e.g., [KH](https://clima.github.io/OceananigansDocumentation/dev/generated/kelvin_helmholtz_instability/)). I'll leave this up to your aesthetics..

Reply:
Okay, I think I was going for what you are suggesting so I'll make that change